### PR TITLE
Remove unused communication model

### PR DESCRIPTION
**Pull request by ragunathjawahar**
https://www.pivotaltracker.com/story/show/165816625

### CODE REVIEW

**Comment by sanchita** [comment]
@ragunathjawahar 
We add the pivotol story link in the description of every PR.
It makes it easier for the reviewer to look-up the requirement.
Also, this makes it easier to find all the PRs related to a story.

Can you please add the same?

**Comment by ragunathjawahar** [comment]
@sanchita added, thanks for the input.

File: app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt [hunk 1]
```diff
@@ -2684,6 +2684,24 @@ class DatabaseMigrationAndroidTest {
     verifyPatientPhoneNumber(dbV42, mobileUuid, "mobile")
     verifyPatientPhoneNumber(dbV42, landlineUuid, "landline")
   }
+
+  @Test(expected = SQLiteException::class)
+  fun drop_communication_table_from_42_to_43() {
```
Comment (vinaysshenoy) [inline app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt]: FYI, we have extension functions on the `SUpportSQLiteDatabase` class which let you verify whether a given table exists or not.

So this entire test can become
```kotlin
val dbv42 = helper.createDatabase(42)
dbv42.assertTableExists("Communication")

val dbv43 = helper.createDatabase(43)
dbv43.assertTableDoesNotExist("Communication")
```

Comment (ragunathjawahar) [reply app/src/androidTest/java/org/simple/clinic/DatabaseMigrationAndroidTest.kt]: Nice!

**Pull request merged**